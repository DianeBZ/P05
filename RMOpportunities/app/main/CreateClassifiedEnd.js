import React, { Component } from 'react';
import { Text, Image, View, StyleSheet,Dimensions,TouchableHighlight,BackAndroid,Navigator} from 'react-native';

import HomePage from './HomePage';
import Translation from './Translation';
var width = Dimensions.get('window').width;
var height = Dimensions.get('window').height;


export default class CreateClassifiedEnd extends Component {

  render() {
	return(
		<View>
			<View style={{height:height*0.08}}/>
			<View style={styles.corps}>
				<Text style={styles.merci}>{Translation[lang].merci}</Text>
				<Image source={require('../img/merci.png')} style={styles.merciImage}/>
				<Text style={styles.merciTexte}>{Translation[lang].mess_merci}</Text>
				<TouchableHighlight onPress={this.onPressAccueil} style={styles.button} underlayColor="rgb(138,183,46)">
					<Text style={styles.buttonText}>{Translation[lang].retour_acc}</Text>
				</TouchableHighlight>
			</View>
		</View>
	);
  }
  // function used to call home page
  onPressAccueil = () => {
    BackAndroid.removeEventListener('Exit',this.onBackAndroid);
    const { navigator } = this.props;
    if (navigator) {
      navigator.push({
        name: 'HomePage',
        component: HomePage,
      })
    }
  };
}

const styles = StyleSheet.create({
	corps:{
		backgroundColor:"#F2F2F2",
		alignItems:'center',
		justifyContent: 'center',
		
	},
	merci:{
		color:"black",
		fontSize:40,
		textAlign:'center',
		marginTop : width*0.15,
	},
	merciImage:{
		height: width*0.4,
		width: width *0.4,
		marginTop : width*0.15,
	},
	merciTexte:{
		color:"black",
		marginTop : width*0.1,
		margin:width*0.05,
		fontSize:18,
	},
	button:{
		marginTop : 5,
		height: width*0.09,
		width: width *0.5,
		borderRadius:5,
		backgroundColor:'#A4D04A',
		justifyContent:'center',
	},
	buttonText:{
      color: "white",
      textAlign: "center",
      fontSize: 20,
	},
});