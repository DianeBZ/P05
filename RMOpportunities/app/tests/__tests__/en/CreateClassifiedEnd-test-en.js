// __tests__/CreateClassifiedEnd-test-en.js
import 'react-native';
import React from 'react';

global.lang = 'en';
import CreateClassifiedEnd from '../../../main/CreateClassifiedEnd';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <CreateClassifiedEnd />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
