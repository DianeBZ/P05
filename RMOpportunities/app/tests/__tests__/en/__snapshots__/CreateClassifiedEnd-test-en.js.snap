// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "height": 106.72,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#F2F2F2",
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 40,
          "marginTop": 112.5,
          "textAlign": "center",
        }
      }
    >
      Thank you !
    </Text>
    <Image
      source={1}
      style={
        Object {
          "height": 300,
          "marginTop": 112.5,
          "width": 300,
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 18,
          "margin": 37.5,
          "marginTop": 75,
        }
      }
    >
      Offer successfully created, it will be controlled by our team as soon as possible.
    </Text>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "backgroundColor": "#A4D04A",
            "borderRadius": 5,
            "height": 67.5,
            "justifyContent": "center",
            "marginTop": 5,
            "width": 375,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "white",
            "fontSize": 20,
            "textAlign": "center",
          }
        }
      >
        Back to home page
      </Text>
    </View>
  </View>
</View>
`;
